code/modules/clothing/suits/armor/bulletproof.dm

```diff
@@ -15,6 +15,27 @@
 	siemens_coefficient = 0.7
 	starting_accessories = list(/obj/item/clothing/accessory/armguards/ballistic, /obj/item/clothing/accessory/legguards/ballistic)
 
+// no accessory
+
+/obj/item/clothing/suit/armor/bulletproof/noacc
```

Comment:
You don't need this full definition, you can just do:
````
/obj/item/clothing/suit/armor/bulletproof/noacc
    starting_accessories = null
````
As written I'm not sure that this item would spawn with no accessories.